wework/src/i18n/locales/zh-CN/common.json

```
@@ -42,6 +42,11 @@
     "im_channel_discord": "Discord",
     "new_project": "新建项目",
     "add_new_project": "添加新项目",
+    "add_local_project": "添加本地项目",
+    "add_remote_project": "添加远程项目",
+    "remote_short": "远程",
+    "new_blank_project": "新建空白项目",
```

Biome (2.5.0)

[error] The key new_blank_project was already declared.

(lint/suspicious/noDuplicateObjectKeys)